### PR TITLE
[DM-2266]: Added information that software item will be deleted on last version deletion.

### PR DESCRIPTION
**Pull request by dafkk**
https://cumulocity.atlassian.net/browse/DM-2266 changes the logic behind the software version deletion. When the last software version gets deleted its software parent item also gets deleted, so there won't be any software without versions when installing a software.

### CODE REVIEW

File: content/device-management-application/managing-device-data-bundle/managing-software.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ The software and all its versions will be deleted from the software repository.
 
 In the details of a specific software, hover over the version entry you want to delete and click the delete icon. The software version will be deleted from the software details.
 
+When the last version in a software is deleted, this will delete its corresponding parent software as well.
```
Comment (rssag) [inline content/device-management-application/managing-device-data-bundle/managing-software.md:79]:
```suggestion
If the last version of a software is deleted, the corresponding parent software is deleted as well.
```

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
what are the target versions?

File: content/device-management-application/managing-device-data-bundle/managing-software.md [hunk 2]
```diff
@@ -76,6 +76,8 @@ The software and all its versions will be deleted from the software repository.
 
 In the details of a specific software, hover over the version entry you want to delete and click the delete icon. The software version will be deleted from the software details.
 
+If the last version of a software is deleted, the corresponding parent software is deleted as well.
```
Comment (bozhidardedov) [inline content/device-management-application/managing-device-data-bundle/managing-software.md:79]: @BeateRixen @rssag We had a discussion while working on https://github.softwareag.com/IOTA/cumulocity-ui/pull/3911 that on the surface there is no hierarchy of software items and we nowhere call anything a "parent software" (the context was the confirmation prompt we initially implemented when deleting the last version).
Can we formulate the sentence by not referring to "parent" software? E.g.:
"If the last version of a software is deleted, the software is entirely removed from the repository."

Comment (dafkk) [reply content/device-management-application/managing-device-data-bundle/managing-software.md:79]: Yep, I was also wondering if this is a good idea. Will remove it :)

**Comment by dafkk** [comment]
> what are the target versions?

Just current develop, not grafting it in any maintenance release.

**Review by rssag — APPROVED** [review]

**Review by bozhidardedov — APPROVED** [review]

**Pull request merged**